net-dialup/freeradius/files/freeradius-3.0.19-systemd-service.patch

```diff
@@ -0,0 +1,48 @@
+--- a/debian/freeradius.service	2019-10-27 11:39:50.194543890 +0100
++++ b/debian/freeradius.service	2019-10-27 11:43:17.352828467 +0100
+@@ -4,9 +4,9 @@
+ Documentation=man:radiusd(8) man:radiusd.conf(5) http://wiki.freeradius.org/ http://networkradius.com/doc/
+ 
+ [Service]
+-Type=notify
+-WatchdogSec=60
+-NotifyAccess=all
++# Type=notify
++# WatchdogSec=60
++# NotifyAccess=all
+ PIDFile=/run/freeradius/freeradius.pid
+ EnvironmentFile=-/etc/default/freeradius
+ 
+@@ -26,17 +26,29 @@
+ # Ensure the daemon can still write its pidfile after it drops
+ # privileges. Combination of options that work on a variety of
+ # systems. Test very carefully if you alter these lines.
+-RuntimeDirectory=freeradius
++RuntimeDirectory=radiusd
+ RuntimeDirectoryMode=0775
+ # This does not work on Debian Jessie:
+ Group=freerad
+ # This does not work on Ubuntu Bionic:
+ ExecStartPre=/bin/chown freerad:freerad /var/run/freeradius
```

Comment:
I realize this line is from upstream, but it's also junk. The permissions/ownership on that directory should be set by the tmpfiles thingy.. and doing it with "chown" is risky as hell regardless. (Aren't we using this patch to change the name from "freeradius" to "radiusd", making this path wrong anyway?)

Reply:
Yeah, my fault, thanks.
I missing some fix available in my previous patch for [3.0.18](https://github.com/gentoo/gentoo/blob/master/net-dialup/freeradius/files/freeradius-3.0.18-systemd-service.patch).

ExecStartPre is wrong yes, I think that in upstream is used because is not used User= but if I remember correctly it's related to something some use case issues.
A default use case could be executed with User and Group defined and for this reason I chose to set these options but there are specific use cases (where I leave the user to customize service) that is needed a boot of radius server as root (probably for DHCP binding). So, this is the reason of the upstream ExecStartPre= command.

However, I will drop it. Thanks for the suggestion.